src/lib/api/api.go

```diff
@@ -119,7 +124,7 @@ func (a *API) Home(w http.ResponseWriter, r *http.Request) {
 	msg := "ping from " + r.RemoteAddr + ", sent pong in monitor"
 	log.Infoln(msg)
 	a.Data = msg
-	a.send(w, a.marshalled())
+	a.send(w, a.Data, "Home")
```

Comment:
So what's the third parameter "Home" for?

Reply:
Yeah, actually it is the `Message` parameter. Should I leave it as an empty string or replace it with `msg` @aquibbaig?

Reply:
Good question! please use nil for most of the cases, unless we need to notify the user for what exactly went wrong. That way to can conditionally check for this message parameter in "send()" or use a varadic function. I can't think of many success cases to notify the users (added route to monitoring on quick input is one of such use cases), let me know if you have sth else in mind.

Reply:
As for now @aquibbaig, I have just added the name of the `function` of API on success.  Should I change them all to `nil`?

Reply:
Yes! That's right. API function as messages don't serve much of a purpose.